packages/downloading-helpers/src/scripts/replays.ts

```diff
@@ -71,8 +71,7 @@ export const getOrFetchReplayArchive = async (
 
     await downloadFile(downloadUrlData.dowloadUrl, replayArchiveFile);
 
-    const zipFile = new Zip(replayArchiveFile);
-    zipFile.extractAllTo(replayDir, /*overwrite=*/ true);
+    await extract(replayArchiveFile, { dir: replayDir });
```

Comment:
I guess `join(getMeticulousLocalDataDir(), "replays", replayId)` could have had existing files. Does 'extract-zip' overwrite by default? (I couldn't tell from the README or tests

Reply:
yep, looks like it overrides by default